docs/helmAgentAddon.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+This doc is used to introduce how to migrate Helm Chart to an Addon.
+
+## Limitations
+1. Not all of build-in Objects of Helm Chart are support in Addon. We only support below:
+* Capabilities.KubeVersion
+* Release.Name
+* Release.Namespace
+
+2. Not support Hooks of Helm Chart currently.
+
+## Migration 
+We have an example for Helm Chart migration in [helloworld_helm](../examples/helloworld_helm).
+1. Copy your helm charts into repo.
```

Comment:
I think you would need to import this into embed.FS

Reply:
done